commands/disconnect.js

```diff
@@ -18,7 +19,7 @@ module.exports = {
 			await interaction.reply({
 				embeds: [
 					new MessageEmbed()
-						.setDescription('I am not currently in the voice channel.')
+						.setDescription(getLocale(defaultLocale, 'DISCORD_BOT_NOT_IN_CHANNEL'))
```

Comment:
I'd like to use `checks.ACTIVE_SESSION` instead, so for now, change the locale to `CHECK_ACTIVE_SESSION`